colelandolt/personal-website
# Initialize Pages Component

## Description

- Relocates templates for the home and about pages into `src/templates/pages`
- Initializes the `pages` component as a Django application
- Creates a new Django model called `Subscriber` which accepts new entries from the associated model form on the home page

## Validation
- `python manage.py makemigrations`
- `python manage.py migrate`
- `python manage.py runserver`
- Submit new entry in the form on the home page
- Log into Django Admin
- Check that the entry was stored in the database
